src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```

Comment:
Classes should have javadoc header comments, to comply with the coding standard.